### ISSUE TITLE
Perceptually uniform `ggplot` colormaps

### ISSUE THREAD

**Issue by asinghvi17** [MakieOrg/MakieThemes.jl#19]
Would be nice to maybe depend on `PerceptualColourMaps.jl` for this.  The current colormaps are just formed by two colors, which I would like to move past in the long term.

**Comment by mkborregaard** [comment]
No reason to depend on that package. Every plot package should depend on `Colorschemes` (and Makie already does this), which also includs all the PerceptualColorMaps under the name "colorcet".

**Comment by asinghvi17** [comment]
Right, but unless I'm missing something, the colormaps here are not included in colorcet, so we'd have to generate them.  They look fine now, this is more of a nice-to-have.

**Comment by mkborregaard** [comment]
Which colormaps are those?
PerceptualColorMaps is a crazy heavy dependency, or at least it used to depend on the entire Images ecosystem

**Comment by asinghvi17** [comment]
Ouch, yeah it depends on Images and PyPlot as well.  Might be best to just generate the colors and write the array out in MakieThemes.

**Comment by mkborregaard** [comment]
Yes that would be my suggestion
Colorschemes tries to be the hub of all things color
